Loop/Managers/Alerts/AlertStore.swift

```diff
@@ -199,10 +231,11 @@ extension AlertStore {
     }
     
     // At the moment, this is only used for unit testing
-    internal func fetch(identifier: Alert.Identifier, completion: @escaping (Result<[StoredAlert], Error>) -> Void) {
+    internal func fetch(identifier: Alert.Identifier? = nil, _ completion: @escaping (Result<[StoredAlert], Error>) -> Void) {
         self.managedObjectContext.perform {
             let storedRequest: NSFetchRequest<StoredAlert> = StoredAlert.fetchRequest()
-            storedRequest.predicate = identifier.equalsPredicate
+            storedRequest.predicate = identifier?.equalsPredicate
```

Comment:
[nit] Consider never using a `nil` predicate with the changes I have pending to increase the cache size to 90 days. That could be a lot of alerts.

Reply:
This function this only used for internal testing.  It intentionally returns all results.